eng/common/scripts/Cadl-Project-Generate.ps1

```diff
@@ -1,3 +1,5 @@
+# For details see https://github.com/Azure/azure-sdk-tools/doc/common/CadlSyncAndGenerateScripts.md
```

Comment:
Those links are not correct they don't include main and they are still the old filename.